openmp/Makefile

```diff
@@ -0,0 +1,39 @@
+include ../Makefile.inc
+UPSTREAM=https://github.com/gcc-mirror/gcc
+CC=$(RUMPRUN_CC)
+CXX=$(RUMPRUN_CXX)
+export CC CXX
+
+all: bin/openmp
+
+bin/openmp:	build/openmp
+	mkdir -p include
+	cp gcc-build/omp.h include/
+	mkdir -p lib
+	cp gcc-build/.libs/libgomp.a lib/
+	rm -rf gcc-build
+	rm -rf gcc-source
```

Comment:
Why is the rm -rf performed here?  It's more polite to leave that choice up to the user to do via `make clean`